test/libsolidity/smtCheckerTests/external_calls/external_inc.sol

```diff
@@ -21,5 +21,6 @@ contract C {
 // SMTIgnoreCex: yes
 // ----
 // Warning 4984: (146-149): CHC: Overflow (resulting value larger than 2**256 - 1) might happen here.
+// Warning 4984: (146-149): CHC: Overflow (resulting value larger than 2**256 - 1) might happen here.
```

Comment:
We get the possible violation twice now. The original one is from calling `inc` directly. The new one is from calling `f` which calls external method, which can call `inc` and potentially cause overflow.

Reply:
We can maybe add a check that the target hasn't been solved yet in CHC itself, before trying to solve it.

Reply:
I am not sure we can do that. It could be that the same assertion in the code is safe when transaction starts in one function, but unsafe when it starts from another, no? In such case we don't want the same version preventing the finding of the unsafe version.

Ah, but we could do that for the targets that has been proven *unsafe* already. Separate PR, probably?

Reply:
Yea, that should work